### PR TITLE
disable arangosearch analyzers in agency

### PR DESCRIPTION
**Pull request by gnusi**
Please note that for legal reasons we require you to sign the 
[Contributor Agreement](https://www.arangodb.com/documents/cla.pdf)
before we can accept your pull requests.

### CODE REVIEW

**Comment by gnusi** [comment]
http://jenkins01.arangodb.biz:8080/view/PR/job/arangodb-matrix-pr/4448/

File: arangod/IResearch/IResearchAnalyzerFeature.cpp [hunk 1]
```diff
@@ -1774,7 +1776,9 @@ arangodb::Result IResearchAnalyzerFeature::remove( // remove analyzer
 }
 
 void IResearchAnalyzerFeature::start() {
-  ApplicationFeature::start();
+  if (!isEnabled()) {
```
Comment (jsteemann) [inline arangod/IResearch/IResearchAnalyzerFeature.cpp]: is it necessary to bail out early in `IResearchAnalyzerFeature::stop()` too?

Comment (gnusi) [reply arangod/IResearch/IResearchAnalyzerFeature.cpp]: @jsteemann good point, I missed it.

**Comment by matthewvon** [comment]
I confirm that agency no longer crashes / SEGFAULTs.

**Pull request merged**